gui/src/main/java/org/verapdf/cli/commands/VeraCliArgParser.java

```diff
@@ -44,6 +43,8 @@
     final static String FIX_METADATA_FOLDER = OPTION_SEP + "savefolder";
     final static String PROFILES_WIKI_FLAG = FLAG_SEP + "pw";
     final static String PROFILES_WIKI = OPTION_SEP + "profilesWiki";
+    final static String REPORT_TYPE = OPTION_SEP + "type";
```

Comment:
About constant name: this is not a report type. it is a processing type.
About constant value: better to use processingType and pt instead of type and t